src/module.ts

```diff
@@ -302,11 +302,15 @@ export default defineNuxtModule<ModuleOptions>({
       nitroConfig.alias = nitroConfig.alias || {}
       nitroConfig.alias['#content/server'] = resolveRuntimeModule('./server')
 
+      const transformers = contentContext.transformers.map((t) => {
+        const name = genSafeVariableName(relative(nuxt.options.rootDir, t)).replace(/_(45|46|47)/g, '_') + '_' + hash(t)
```

Comment:
What does this replace do `.replace(/_(45|46|47)/g, '_')?`

Reply:
There are common charachters to simplify human redable part of name. Hash ensures being unique.